modules/cluster-logging-rn-5.4.14.adoc

```diff
@@ -0,0 +1,18 @@
+//module included in cluster-logging-release-notes.adoc
+:content-type: REFERENCE
+[id="cluster-logging-release-notes-5-4-14{context}"]
+= Logging 5.4.14
+This release includes link:https://access.redhat.com/errata/RHBA-2023:1843[OpenShift Logging Bug Fix Release 5.4.14].
```

Comment:
Are you sure this link is correct? It does not open for me, others open for me. I will approve once I get your confirmation re link.

Reply:
Yep, not live yet.